nasa/ow_simulator
# Oceanwater 968 allow arm to continue moving on arm faults

## Linked Issues:
| EPIC ⚡ | [OCEANWATER-967](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-967) |
| :----------- | :----------- |
| Jira Ticket 🎟️ | [OCEANWATER-968](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-968) |
| Github :octocat:  | # |


## Summary of Changes
* Added option of continuing arm motion in faults

## Test
* roslaunch ow atacama_y1a.launch 
*  In a separate terminal check rosparam get /faults/continue_arm_in_fault - it should be set to false. You can change it to true using command line rosparam set /faults/continue_arm_in_fault true or using  rqt check the following box.
rqt > Dynamic Reconfigure > faults > continue_arm_in_fault
* use ros action to command any motion example ./unstow_action_client.py 
* Insert a fault using rqt 
* rqt > Dynamic Reconfigure > faults > shou_yaw_joint_locked_failure

observe the difference in behavior with stopping the arm in fault or allowing the motion.
